linera-sdk/src/lib.rs

```diff
@@ -38,6 +38,7 @@
 #![deny(missing_docs)]
 
 pub mod base;
+#[allow(missing_docs)]
```

Comment:
Minor: I think turning off the lint here might be too broad. I realized that there are `wit_types` sub-modules in `contract` and `service` which also have `#[allow(missing_docs)]` (so I guess I have run into this before :sweat_smile:). Could we do the same approach here, and have `wit_system_api` sub-modules with the attribute?

Reply:
Done; please let me know if this is what you had in mind!